addons/hw_drivers/iot_handlers/drivers/PrinterDriver_L.py

```diff
@@ -172,11 +172,22 @@ def send_status(self):
         }
         event_manager.device_changed(self)
 
-    def print_raw(self, data):
-        process = subprocess.Popen(["lp", "-d", self.device_identifier], stdin=subprocess.PIPE)
+    def print_raw(self, data, landscape=False, duplex=True):
```

Comment:
is there another PR altering the functional calls ?

Reply:
not for the moment